.travis.yml

```diff
@@ -1,5 +1,4 @@
 language: node_js
 node_js:
   - "0.12"
-  - "0.11"
-  - "0.10"
+  - "5"
```

Comment:
I see that `"node"` is an alias for the latest stable version, so I just switched to that.